tests/test_run_vid_diffusion_online.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+from itertools import product
+
+import pytest
+import torch.multiprocessing as mp
+
+sys.path.append(sys.path[0] + "/..")
+import train
+from data import create_dataset_temporal
+from options.train_options import TrainOptions
+from scripts.gen_vid_diffusion import InferenceDiffusionOptions, inference
+
+json_like_dict = {
+    "name": "joligen_utest",
+    "output_display_env": "joligen_utest",
+    "output_display_id": 0,
+    "gpu_ids": "0",
+    "data_load_size": 64,
+    "data_crop_size": 64,
+    "data_online_creation_crop_size_A": 64,
+    "data_online_creation_crop_delta_A": 50,
+    "data_online_creation_mask_delta_A_ratio": [[0.2, 0.2]],
+    "data_online_creation_crop_size_B": 64,
+    "data_online_creation_crop_delta_B": 50,
+    "train_n_epochs": 1,
+    "train_n_epochs_decay": 0,
+    "data_max_dataset_size": 10,
+    "data_relative_paths": True,
+    "train_G_ema": True,
+    "dataaug_no_rotate": True,
+    "G_unet_mha_num_head_channels": 32,
+    "G_unet_mha_channel_mults": [1, 2, 4, 8],
+    "G_nblocks": 9,
+    "G_padding_type": "reflect",
+    "data_online_creation_rand_mask_A": True,
+    "train_export_jit": False,
+    "train_save_latest_freq": 10,
+    "G_diff_n_timestep_test": 10,
+    "train_batch_size": 1,
+    "data_temporal_number_frames": 8,
+    "data_temporal_frame_step": 1,
+    "G_diff_n_timestep_train": 6,
```

Comment:
Beware I don´t believe you can theoretically have timestep_test < timestep_train.

Reply:
maybe I misunderstood opt.G_diff_n_timestep_train is 2000 and  opt.G_diff_n_timestep_test is 1000 in defalut setting ?

Reply:
I had overwritten "G_diff_n_timestep_test", it is corrected now.